Sources/TuistSupport/Logging/StandardLogHandler.swift

```diff
@@ -41,7 +43,9 @@ public struct StandardLogHandler: LogHandler {
                 case .error:
                     string = message.description.red()
                 case .warning:
-                    string = message.description.yellow()
+                    string = nil
+                    shouldOputput = false
+                    WarningController.shared.append(warning: message.description)
```

Comment:
You can do an early return here and remove the `shouldOutput` variable completely

Reply:
Good point! I adjusted it.